prov/verbs/src/fi_verbs.h

```diff
@@ -984,50 +986,61 @@ vrb_send_iov_flags(struct vrb_ep *ep, struct ibv_send_wr *wr,
 		      uint64_t flags)
 {
 	size_t len = 0;
-	enum fi_hmem_iface iface;
-	uint64_t device;
+	enum fi_hmem_iface iface = FI_HMEM_SYSTEM;
+	uint64_t device = 0;
 	void *bounce_buf;
-	void *send_desc;
+	void *send_desc = NULL;
```

Comment:
It looks like these assignments are only needed in the hmem_enabled case.  If that's the case, we should only pay the cost of setting these when enabled is true.

Reply:
Will fix.